L1Trigger/L1TGlobal/src/GlobalBoard.cc

```diff
@@ -643,6 +650,26 @@ void l1t::GlobalBoard::runGTL(const edm::Event&,
           }
           //                    delete eSumZdcCondition;
 
+        } break;
+        case CondAXOL1TL: {
+          AXOL1TLCondition* axol1tlCondition = new AXOL1TLCondition(itCond->second, this);
+
+          axol1tlCondition->setVerbosity(m_verbosity);
+
+          axol1tlCondition->setModelVersion(m_axol1tlModelVersion);
+
+          axol1tlCondition->evaluateConditionStoreResult(iBxInEvent);
+
+          cMapResults[itCond->first] = axol1tlCondition;
```

Comment:
Okay, I've been doing a bit more research here. I was concerned in a previous comment that this leaks memory, however **I am fairly confident that this doesn't leak**. After making a `new AXOL1TLCondition` the overall pointer gets handed off to `cMapResults`.

Digging a little deeper, this `cMapResults` comes from here: https://github.com/cms-sw/cmssw/blob/245daa4d7ad8e1412084c2d66381edfd8835d7ad/L1Trigger/L1TGlobal/src/GlobalBoard.cc#L546-L548
As a reference to an underlying variable `m_conditionResultMaps[iChip]`

if I look around then for `m_conditionResultMaps` I find all secondary indexes get deleted here: https://github.com/cms-sw/cmssw/blob/245daa4d7ad8e1412084c2d66381edfd8835d7ad/L1Trigger/L1TGlobal/src/GlobalBoard.cc#L972-L980